doc/GuideForCommitters.md

```diff
@@ -0,0 +1,87 @@
+<!--
+Copyright (c) 2019, 2019 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+
+# Eclipse OpenJ9 Committer Guide
+
+This document outlines some general rules for Eclipse OpenJ9 committers to
+follow when reviewing and merging pull requests and issues.  It also provides
+a checklist of items that committers must ensure have been completed prior to
+merging a pull request.
+
+
+## General Guidelines
+
+* Committers should not merge their own pull requests.
+
+* Do not merge a pull request if its title is prefixed with `WIP:` (indicating
+it's a work-in-progress)
+
+* If a review or feedback was requested by the author by `@mentioning` particular
+developers in a pull request, do not merge until all have provided the input
+requested.
+
+* Do not merge a pull request until all validation checks and continuous
+integration builds have passed.  If there are circumstances where you believe
+this rule should be violated (for example, a known test failure unrelated to
+this change) then the justification must be documented in the pull request
+prior to merging.
+
+* If a pull request is to revert a previous commit, ensure there is a reasonable
+explanation in the pull request from the author of the rationale along with a
+description of the problem if the commit is not reverted.
```

Comment:
We should also request committers open an issue that links the revert commit to the original PR and tags the original author.  This will help ensure that the cause of the revert is addressed and not lost.